### PR TITLE
Omit debuginfo files from the main SDK archive

### PR DESCRIPTION
**Pull request by keithc-ca**
Most of the omitted files are available in the debug image (an exception is `*.map` files on Windows).

Fixes: #9996.

### CODE REVIEW

**Comment by keithc-ca** [comment]
Jenkins test sanity aix,osx,plinux,xlinux,zlinux,win jdk11

File: buildenv/jenkins/common/build.groovy [hunk 1]
```diff
@@ -288,7 +288,11 @@ def archive_sdk() {
         def testDir = "test"
 
         dir(OPENJDK_CLONE_DIR) {
-            sh "tar -C ${buildDir} -zcvf ${SDK_FILENAME} ${JDK_FOLDER}"
+            // The archiver receives pathnames on stdin and writes to stdout.
+            def archiveCmd = SPEC.contains('zos') ? 'pax -wvz' : 'tar -cvz -T -'
+            // Filter out unwanted files (most of which are available in the debug-image).
+            def filterCmd = "sed -e '/\\.dbg\$/d' -e '/\\.debuginfo\$/d' -e '/\\.diz\$/d' -e '/\\.dSYM\\//d' -e '/\\.map\$/d' -e '/\\.pdb\$/d'"
```
Comment (DanHeidinga) [inline buildenv/jenkins/common/build.groovy:294]: There's a `//` before the `d` in the `.dSYM` case:
`-e '/\\.dSYM\\//d'` -> `-e '/\\.dSYM\\/d'`

Comment (keithc-ca) [reply buildenv/jenkins/common/build.groovy:294]: That's intentional (to match `.dSYM/`).

Comment (DanHeidinga) [reply buildenv/jenkins/common/build.groovy:294]: And looking at the osx build, it's clearly working

**Pull request merged**